src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -2,8 +2,9 @@
 
 import json
 
+from qblox_instruments.native.generic_func import SequencerStates
 from qblox_instruments.qcodes_drivers.cluster import Cluster as QbloxCluster
-from qblox_instruments.qcodes_drivers.qcm_qrm import QcmQrm as QbloxQrmQcm
+from qblox_instruments.qcodes_drivers.module import Module as QbloxModule
```

Comment:
Why not just `Module`?
```suggestion
from qblox_instruments.qcodes_drivers.module import Module
```

Reply:
No particular reason. We don't have anything of our own called Module anyways, so should not be ambiguous